README.md

```diff
@@ -39,17 +39,17 @@ npm i -g @tinystacks/ops-cli;
 This package includes other sample dashboard as well. As an example, it includes a sample dashboard that has ECS and AWS account info. To use that, follow these steps:
 
 ```
-curl https://raw.githubusercontent.com/tinystacks/ops-cli/main/samples/ecs-dashboard-sample.yml -o ecs-dashboard-sample.yml
+curl https://raw.githubusercontent.com/tinystacks/opsconsole/main/samples/ecs-dashboard-sample.yml -o ecs-dashboard-sample.yml
```

Comment:
This shouldn't change unless we are going to update the repo name which I don't think we plan to.